### ISSUE TITLE
[Feature-Request] allow grant permission on column level

### ISSUE THREAD

**Issue by hiepxanh** [pgmodeler/pgmodeler#1716]
**Feature description**
It cannot find the document about allow permission on specific column
so I want we have text box that allow me to write the column name I want 
**Sample image**
![image](https://user-images.githubusercontent.com/16970990/191931176-c270aa91-0e24-4fe1-89b5-52ae61852fad.png)

![image](https://user-images.githubusercontent.com/16970990/191931298-79e9e1de-1516-4975-a858-b7dbd7f0f22c.png)

**Additional info**

**Comment by rkhaotix** [comment]
Column level permissions can be configured directly from column editing form or focusing a column in design view, right click it and trigger Quick > Edit permissions

https://www.pgmodeler.io/support/docs/325-permissions?v=0.9.4

**Comment by hiepxanh** [comment]
wow really? that is really amazing, I doesn't notice that, how can I contribute a little in the docs to make it more clearly?

**Comment by rkhaotix** [comment]
I'll try to make a deep revision in the docs once 1.0 stable is out.

**Comment by hiepxanh** [comment]
Yes, thank you so much, that will help alot, I will leave this issue open :D
